requirements/prod-requirements.txt

```diff
@@ -106,12 +106,12 @@ markdown==2.2.1
 markupsafe==1.0
 mock==2.0.0
 msgpack-python==0.5.6
-ndg-httpsclient==0.5.0
-newrelic==3.4.0.95
+ndg-httpsclient==0.5.1
+newrelic==4.0.0.99
```

Comment:
@snopoke do we still need newrelic installed? If not we can just kill this dependency